### ISSUE TITLE
"outdated" files were deleted from cache without user interaction

### ISSUE THREAD

**Issue by SeriousBarbie** [OldUnreal/UnrealTournamentPatches#1706]
I removed UnrealTournament.ini temporary to get a clean one for a bug report. I noticed that a lot of "outdated" files were deleted on first UT start without asking or my permission - that's a no go! 👎 Sadly I didn't have a backup of these.

User should be asked before doing such unexpected operations.

**Comment by SeriousBuggie** [comment]
It is part of cache management system. When you remove ini file, it use defaults. So it effectively change your cache settings.
Which by default clear by date.
So on run, game wipe all old entries from cache.
Default.ini:
```ini
[Core.System]
PurgeCacheDays=30
```
So no any mistake here.
Wipe ini in general not safe operation with side effects,

**Comment by SeriousBarbie** [comment]
I didn't say that it is a mistake,  it is just odd behaviour.
Why do I rely on old cache entries? Sometimes on visited servers I notice good effects, music, maps, whatever and think: "ok, that's in cache and can be investigated later." - "Boooh, where is it now?"

**Comment by SeriousBuggie** [comment]
Game can use files which not listed in cache.ini. So not need sync them. At least in direction from cache.ini to files.

PurgeCacheDays is old option. It always be default 30.

You can alter default.ini value, but any patch overwrite default.ini.

**Comment by Chamberly** [comment]
I've repeatedly asked to put this to 0 in the past. Did not make it through. :(

**Comment by stijn-volckaert** [comment]
I agree this is a terrible default value. I've changed the default value to 0, so this will no longer happen in 469e RC5 and later patches
